Sources/PackageCollectionsModel/Formats/v1.md

```diff
@@ -0,0 +1,237 @@
+# Package Collection
+
+Package collections are short, curated lists of packages and associated metadata that can be imported
+by SwiftPM to make package discovery easier. Educators and community influencers can publish
+package collections to go along with course materials or blog posts, removing the friction of using
+packages for the first time and the cognitive overload of deciding which packages are useful for
+a particular task. Enterprises may use collections to narrow the decision space for their internal
+engineering teams, focusing them on a trusted set of vetted packages.
+
+## Creating a Package Collection
+
+A package collection is a JSON document that contains a list of packages and metadata per package.
+
+To begin, define the top-level metadata about the collection:
+
+* `name`: The name of the package collection, for display purposes only.
+* `overview`: A description of the package collection. **Optional.**
+* `keywords`: An array of keywords that the collection is associated with. **Optional.**
+* `formatVersion`: The version of the format to which the collection conforms. Currently, `1.0` is the only allowed value.
+* `revision`: The revision number of this package collection. **Optional.**
+* `generatedAt`: The ISO 8601-formatted datetime string when the package collection was generated.
+* `generatedBy`: The author of this package collection. **Optional.**
+    * `name`: The author name.
+* `packages`: An array of package objects.
+
+#### Add packages to the collection
+
+Each item in the `packages` array is a package object with the following properties:
+
+* `url`: The URL of the package. Currently only Git repository URLs are supported.
+* `summary`: A description of the package. **Optional.**
+* `keywords`: An array of keywords that the package is associated with. **Optional.**
+* `readmeURL`: The URL of the package's README. **Optional.**
+* `license`: The package's *current* license information. **Optional.**
+    * `url`: The URL of the license file.
+    * `name`: License name. [SPDX identifier](https://spdx.org/licenses/) (e.g., `Apache-2.0`, `MIT`, etc.) preferred. Omit if unknown. **Optional.**
+* `versions`: An array of version objects representing the most recent and/or relevant releases of the package.
+
+#### Add versions to a package
+
+A version object has metadata extracted from `Package.swift` and optionally additional metadata from other sources:
+
+* `version`: The semantic version string.
+* `manifests`: A map of manifests by tools version. The keys are (semantic) tools version (more on this below), while the values are:
```

Comment:
New

Reply:
Maybe it would be good to say "A map of manifests by Swift tools version" here.  I know it's clear later but only because it comes just after the previous "version" key, just to be super-sure nobody mistakes this for the package version (even though it does say "tools").